.github/workflows/publish.yml

```diff
@@ -0,0 +1,38 @@
+name: publish
+
+on:
+  release:
+    types: [published]
+
+permissions:
+  contents: read
+
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v3
+      - name: Set up Python
+        uses: actions/setup-python@v3
+        with:
+          python-version: '3.10'
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          pip install build
+      - name: Build package
+        run: python -m build
+      - name: Publish package
+        uses: pypa/gh-action-pypi-publish@release/v1
+        with:
+          user: __token__
+          password: ${{ secrets.PYPI_API_TOKEN }}
+
+      - uses: actions/setup-node@v3
+        with:
+          node-version: 18
+      - name: publish npm
+        run: |
+          cd js
+          npm publish
```

Comment:
The `js` seems to use the `yarn` package manager, not `npm`.

Reply:
https://classic.yarnpkg.com/lang/en/docs/cli/publish/

Reply:
Are yarn published packages still findable by colab?